src/main/java/touro/chess/BishopPiece.java

```diff
@@ -5,6 +5,8 @@
 
 public class BishopPiece extends AbstractPiece
 {
+    private final int score = 3;
```

Comment:
You added scores, but you don't actually use it.
`whoIsWinning()` should add up the scores.
Provider another method that returns who has the most influence.